go.mod

```
@@ -19,7 +19,7 @@ retract (
 replace (
 	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf // v2.6
 	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20180202092358-40e2722dffea
-	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190104202606-0ac367fd6bee+incompatible
+	github.com/docker/distribution => github.com/distribution/distribution v2.8.0+incompatible
```

Adding `v2` at the end should remove the need for `+incompatible`. See [+incompatible versions](https://go.dev/ref/mod#incompatible-versions) on the Go modules reference document
```suggestion
	github.com/docker/distribution => github.com/distribution/distribution/v2 v2.8.0
```

---

Good point